src/librustdoc/html/render/cache.rs

```diff
@@ -574,6 +574,10 @@ fn build_index(krate: &clean::Crate, cache: &mut Cache) -> String {
     // has since been learned.
     for &(did, ref item) in orphan_impl_items {
         if let Some(&(ref fqp, _)) = paths.get(&did) {
+            if item.name.is_none() {
+                // this is most likely from a typedef
+                continue;
+            }
```

Comment:
What is this change for? I can't see anything in this PR that would require this.

Reply:
It's not needed anymore, indeed! It was part of my debugging.